### ISSUE TITLE
Running command does not output loading information, and not colorful

### ISSUE THREAD

**Issue by chbybnwr** [ksxnodeapps/package-alt-cson#1]

**Comment by chbybnwr** [comment]
For example when installing packages, there is no spinning stuff
Awesome package anyway, since I am now obsessed writing everything in coffeescript and cson

**Comment by KSXGitHub** [comment]
There're some ways to get around this:
1. Use `script` command (which is not always available in Windows)
2. Use pure `bash` script instead of JavaScript (again, not always available in Windows, unless user installs Git Bash from git-scm.com)

**Comment by chbybnwr** [comment]
When you said script, did you mean npm script?

**Comment by KSXGitHub** [comment]
No, I mean UNIX/Linux [`script`](http://man7.org/linux/man-pages/man1/script.1.html) command

**Comment by chbybnwr** [comment]
Could you provide some example?

**Comment by KSXGitHub** [comment]
@chabib 

Without `script`, loading information and color only in real tty, they won't be in pipe

```javascript
const {stdout, stderr} = child_process.spawnSync('npm', ['install'])
process.stdout.write(stdout)
process.stderr.write(stderr)
```

Invoking commands via `script`, on the other hand, would display all information regardless of whether stdio is tty or pipe. (`script` tells the program that its output is not a pipe but a tty)

```javascript
const {stdout, stderr} = child_process.spawnSync('/usr/bin/script', ['-c', 'npm', 'install'])
process.stdout.write(stdout)
process.stderr.write(stderr)
```

`package-alt-cson` and `package-alt-yaml` don't display loading information because they execute `npm` via a pipe, causing `npm` to print neither loading information nor color.

#### Terminology

* **TTY:** Terminal Screen (sort of)
* **pipe:** Output to another program's input
